CHANGELOG.md

```diff
@@ -1,6 +1,28 @@
 # Release notes
 This page documents production updates to Cloud Tools for IntelliJ. You can check this page for announcements about new or updated features, bug fixes, known issues, and deprecated functionality.
 
+## 17.9.1
+
+### Added
+  - Added the ability to change the name of the staged artifact for App Engine flexible deployments.
+    [1151](https://github.com/GoogleCloudPlatform/google-cloud-intellij/issues/1151)
```

Comment:
is the link here mixed up with the link to the first bullet under 'changed'?

Reply:
I'm not sure what you mean; do you mean does the line-break cause an issue with formatting? Or do you mean are the links incorrectly swapped between this bullet and the next?

Reply:
yea sorry - that the links are swapped between the two

Reply:
My original intention was that the addition of the field points to the tracking bug of the entire feature (#1151) and the default config change point to the PR that made it happen (#1611), but I can see how it's confusing. I can switch them, WDYT?

Reply:
Seems to me like you should swap them and instead of pointing to the PR, point to the issue.

Reply:
Done.